kernel-switcher/switch_kernel.py

```diff
@@ -140,12 +140,55 @@ def parse_args(argv):
         help="do not write anything to the grub config",
     )
 
-    args = parser.parse_args(argv[1:])
-    return args.kernel[0], args.dry_run
+    parser.add_argument(
```

Comment:
Question: why this an `enable-XX`? If this is not specified the machine will be un-reprovisionable via MAAS if switching to a real time kernel, is there a situation where we don't want this?

Reply:
Yes, we don't need it on projects handled by other methods of provisioning, where indeed efi runtime might affect realtime-ness.